daycare/resources/daycare_static_users.py

```diff
@@ -0,0 +1,65 @@
+USERNAME = 'staticUser'
```

Comment:
as I mentioned in #65 , I don't really like this solution, as it'll be better to mention those explicitly and in one place, instead of following loops through two different files.

Reply:
The file `daycare_static_users` with the static information is very cluttered, so if all this information sat in only one place as you mentioned, would it not be in this case that it creates more clutter, and is less readable and understandable?
I would love to know what you mean by saying that it is best to have everything in one place, do you mean that it should be in the data migration file (that will include the all static data), or in the `daycare_static_users.py` file the will include all the logic and the loops that are in the data migration right now?

Reply:
Think about the (new) developer experience, trying to either create or understand a test using some of the daycares created from this information. 
since it is all created within loops, he'll have to match the different parameters according to their indexes to understand what he should expect. e.g: which email goes with which username, pass, name, etc...
this is not readable.
I much prefer having a longer and more cluttered file with a list of each daycare object parameters, so I'll understand in one look what I see and which daycare has which property.

Reply:
And what about if I will transfer all the static data from `resources/daycare_static_users.py` to the data migration,
and there I will still use the loop, that each static daycare user, will get his data from the lists, that each list is a field in the model.
For example (for the name field):
DAYCARE_NAME = [name1, name2, ...., ...nameN]
and I will run in a loop for N static users, and for each user: daycare.name = DAYCARE_NAME[i].
I think in that way it will be readable and easy to understand for the reader, instead of creating N lists, that each list will include all the fields of the user (that will be a little messy).
@Omeramsc  tell me what u think

Reply:
Got your point.
So I created the static daycare users only in the data migrations,
but what do you think about the static images data? It does not really matter what pictures each static user will get so I thought it would be right to leave the situation as it is, I would love to know what you think about it.

Reply:
It doesn't matter now, but what about the future?
anyway, don't use any "random" element in tests. even if it doesn't matter. tests has to be reliable, with specific data expected in and out.

Reply:
I don't think it will matter in the future also, because this data is static, and has no effect on future "real" users that will log in to the website.
about the random images, i will change it.